compiler_opt/tools/extract_ir.py

```diff
@@ -59,11 +59,15 @@
     'Include only those modules with a command line matching this regexp. '
     'Setting it to None for not filtering. Note that the regexp is applied '
     'independently for each separate command line option. For example, ^-Oz$ '
-    'will match Oz - built binaries.')
+    'will match Oz - built binaries. Does not work with lld_thinlto_build.')
 flags.DEFINE_bool(
     'thinlto_build', False, 'Set if the build was ThinLTO, to '
     'ensure index files are also copied. The build is assumed to have had'
     '-mllvm -lto-embed-bitcode=post-merge-pre-opt passed to clang.')
+flags.DEFINE_bool(
```

Comment:
rather make the thinlto_build an enum with 3 values: None, distributed (i.e. clang-based), in_process (i.e. lld-based). Otherwise it suggests the 2 flags are orthogonal, and that's not the case.